GVFS/GVFS.FunctionalTests/Should/FileSystemShouldExtensions.cs

```diff
@@ -297,24 +343,27 @@ public DirectoryInfo WithAttribute(FileAttributes attribute)
                             if (Directory.GetFileSystemEntries(expectedEnumerator.Current.FullName, "*", SearchOption.TopDirectoryOnly).Length == 0)
                             {
                                 expectedMoved = expectedEnumerator.MoveNext();
+
                                 continue;
                             }
                         }
 
                         Assert.Fail($"File names don't match: expected: {expectedEnumerator.Current.FullName} actual: {actualEnumerator.Current.FullName}");
+                        expectedMoved = expectedEnumerator.MoveNext();
+                        continue;

```

Comment:
Why was this change needed?  With the call to `Assert.Fail` on line 351 will we ever hit this code?

Reply:
Not sure why @derrickstolee added this.  Seems unnecessary but maybe debugging you can continue through the `Fail`?

Reply:
I think I had changed the assert to be something that concatenated the error (so I could see the full list of differences) but somehow forgot to revert this change.

Reply:
I think I had changed the assert to be something that concatenated the error (so I could see the full list of differences) but somehow forgot to revert this change.